code/game/objects/items/plushes.dm

```diff
@@ -788,3 +788,11 @@
 		'sound/effects/slosh.ogg' = 1,
 		'sound/effects/splat.ogg' = 2
 	)
+/obj/item/toy/plush/shark
+	name = "shark plushie"
+	desc = "A plushie depicting a somewhat cartoonish shark. The tag calls it a 'hákarl', noting that it was made by an obscure furniture manufacturer in old Scandinavia. Popular with cute girls who wear striped socks."
```

Comment:
I feel like the reference could be made better, something like adopting the same joke as ULTRAKILL ("has been considered a status symbol in human society, possibly as a sign of femininity or "those who receive", though the details have been lost to time.") or making a programmer joke instead, what with programming socks

Reply:
It's a thin line, because I want it to be interesting and obvious without being ham-fisted. I feel like this gets the point across without being too much.

Reply:
I think this is fine

Reply:
The main points I wanted to get across were that it was an in-universe expy of a Blahaj and trans girls like it.